databroker/tests/test_config.py

```diff
@@ -46,7 +46,25 @@
 
 
 def test_from_config():
-    Broker.from_config(EXAMPLE)
+    broker = Broker.from_config(EXAMPLE)
+    config = broker.get_config()
+    print(config)
+    # we explicitly test for parts we know should be accepted
+    mds_example = EXAMPLE['metadatastore']['config']
+    reg_example = EXAMPLE['assets']['config']
+    root_map_example = EXAMPLE['root_map']
+
+    mds_config = config['metadatastore']
+    reg_config = config['assets']
+    root_map_config = config['root_map']
+
+    for key, val in mds_config.items():
```

Comment:
This check won't catch the case where `config` is missing an item. It's probably simpler to just do `assert mds_config == mds_example`. Pytest provides some introspection that will give an element-by-element view of any differences, which I think the goal of the way you have factored this here.

Reply:
ah i was worried this would be false:
```py
In [1]: a = dict(a=1)

In [2]: b = dict(a=1)

In [3]: a==b
Out[3]: True
```

thanks ill add it in